SUMMARY.md

```diff
@@ -24,14 +24,13 @@
    * [Vector calculus limitations](coding-the-legislation/25_vectorial_computing.md)
    * [Case disjunction](coding-the-legislation/30_case_disjunction.md)
    * [Periods](coding-the-legislation/35_periods.md)
+   * [Parameters](coding-the-legislation/legislation_parameters.md)
```

Comment:
Not sure this fits well here. The pages starting by `10`, `30`, etc. are a tutorial that is homogenous. Introducing a new page is a middle without thinking the whole flow is not great.

As we don't have a clear idea of how to organize the doc, maybe we can just put this page a little further, at the same level than `Writing YAML tests` ?